docs/getting-started.md

```diff
@@ -153,6 +157,23 @@ We also need to configure sass-loader to understand the `@material` imports used
 }
 ```
 
+Then we need to configure autoprefixer together with PostCSS. The plugin adds vendor-specific styles in the Sass files.  Add `autoprefixer` on top of your `webpack.config.js`:
+
+```js
+const autoprefixer= require('autoprefixer');
+```
+
+Change `{ loader: 'postcss-loader' }` within the same file to:
+
+```js
+{
+  loader: 'postcss-loader',
+  options: {
+     plugins: () => [autoprefixer({ browsers: ['last 2 versions'] })]
```

Comment:
`last 2 versions` is part of the default behavior for autoprefixer, and MDC Web builds using the default, so I think we can omit `browsers`.

Instead, add `grid: false` in order for the Layout Grid package to work as intended (see https://github.com/material-components/material-components-web/pull/228#discussion_r98685522). We might want to add a "Note:" blockquote below this code to explain why.